### DISCUSSION TITLE
Error when using sim_post

### DISCUSSION THREAD

**Discussion by chrisberglund** [ianjonsen/foieGras#61] [Q&A]
Hello,

I'm trying to use the sim_post on some Weddell seal data to generate simulated tracks for multiple imputation. The function works great for nearly all of my seals, but I do get an error for four of them, and I haven't been able to figure out why. When I try using sim_post for those seals, I get

Error in solve.default(jp) : 
  system is computationally singular: reciprocal condition number = 1.3823e-17

I was wondering if you had any suggestion of what issue I should be looking for?

**Comment by chrisberglund** [comment]
Of course, the moment I post this, I figure out that it has to do with some rather problematic time gaps in positions that I should have caught earlier.

**Reply by ianjonsen** [reply]
Thanks for letting me know about this. If you have time to send a reproducible example then I could use it to implement a more informative error msg. Otherwise, good to know you resolved the issue yourself!
